lib/buildkite/test_collector/ci.rb

```diff
@@ -31,13 +31,14 @@ def analytics_env
       "url" => ENV["BUILDKITE_ANALYTICS_URL"],
```

Comment:
@blaknite is this `:key` going to work for our needs? Is this a static value?

Reply:
My rudimentary understanding is that CircleCI has "workflows" and "workflow instances" which are synonymous with "pipelines" and "builds". If my understanding is correct, `CIRCLE_WORKFLOW_ID` is exactly the variable we're after.

Does my understanding match yours?